resources/quality/tizyx_k25/tizyx_k25_normal.inst.cfg

```diff
@@ -13,17 +13,17 @@ global_quality = True
 adhesion_type = skirt
 skirt_line_count = 2
 skirt_gap = 2
-cool_fan_speed_0 = 100
 fill_outline_gaps = True
-infill_angles = [0,90 ]
+infill_angles = []

```

Comment:
`infill_angles` and `skin_angles` are empty `[]` by default, so you can just well remove them